framework/include/utils/MooseUtils.h

```diff
@@ -678,18 +680,19 @@ void MaterialPropertyStorageDump(
  * COLOR_DEFAULT
  *
  * Also handles single line color codes
- * COLOR_CYAN sub_app: 0 Nonline |R| = COLOR_GREEN 1.0e-10 COLOR_DEFAULT
+ * COLOR_CYAN sub_app: 0 Nonlinear |R| = COLOR_GREEN 1.0e-10 COLOR_DEFAULT
  *
  * Not indenting the first line is useful in the case where the first line is actually finishing
  * the line before it.
  */
 void indentMessage(const std::string & prefix,
                    std::string & message,
                    const char * color = COLOR_CYAN,
-                   bool dont_indent_first_line = true);
+                   bool dont_indent_first_line = true,
+                   const std::string & post_prefix = ": ");
```

Comment:
I don't love this parameter addition. Are we ever going to need to add a `post_post_prefix`?

Reply:
no with this we control all the characters in the string. 

It's hardcoded to be a ":" because of multiapps, but [DBG]: doesnt look that good